pkg/lib/operation/project/local/template/upgrade/operation.go

```diff
@@ -25,6 +36,163 @@ type dependencies interface {
 	EncryptionApi() (*encryptionapi.Api, error)
 }
 
-func Run(projectState *project.State, tmpl *template.Template, o Options, d dependencies) (string, error) {
-	return "", fmt.Errorf("not implemented")
+func Run(projectState *project.State, tmpl *template.Template, o Options, d dependencies) error {
+	logger := d.Logger()
+
```

Comment:
Implementacia, podobna ako `use`, ale riesi sa `add/change/delete`, takze je to nahackovane este viac :joy:

Reply:
👍 😃